starter-files/starter.html

```diff
@@ -189,15 +189,15 @@ <h5 class="text-sm">
         Message Input
       </p>
 
-      <button class="text-3xl text-grey border-r-2 border-grey p-2" aria-label="File menu" type="button">
+      <button class="text-3xl text-grey border-r-2 border-grey p-2" aria-label="add a file to this message">
```

Comment:
@MelSumner per our discussion this morning, I think it's safe to strip the `type="button"` I had here. Please let me know if I should put it back

Reply:
Correct!